ext/standard/string.c

```diff
@@ -3863,6 +3866,38 @@ PHPAPI zend_string *php_addcslashes_str(const char *str, size_t len, const char
 }
 /* }}} */
 
+/* {{{ php_repr_str */
+PHPAPI zend_string *php_repr_str(const char *str, size_t len) {
```

Comment:
Ah, and I would also improve the naming. I don't find this particularly descriptive.

Reply:
Any suggestions about the naming? basically this function take a string as input, returns a zend_string which converts all non-printable chars to a human readable format (e.g. `chr(256)` will be convert to `"\xff"`).

The `repr` part in the function name was inspired by the python [`repr` function](https://docs.python.org/3/library/functions.html#repr).

> `repr` returns a string containing a printable representation of an object in Python.